content/main.js

```diff
@@ -390,16 +390,24 @@ var tblatex = {
       div.parentNode.removeChild(div);
   }
 
-  function replace(string, pattern, replacement) {
+  function replace_marker(string, replacement) {
+    var marker = "__REPLACE_ME__";
+    var oldmarker = "__REPLACEME__";
+    var l = marker.length;
     var log = "";
-    var i = string.indexOf(pattern);
+    var i = string.indexOf(marker);
     if (i < 0) {
-      log += "\n!!! Could not find the place marker '__REPLACE_ME__' in your template.\n";
-      log += "This would be the place, where your LaTeX expression is inserted.\n";
-      log += "Please edit your template and add this marker.\n";
-      return [, log];
+      i = string.indexOf(oldmarker);
+      if (i < 0) {
+        log += "\n!!! Could not find the place marker '" + marker + "' in your template.\n";
+        log += "This would be the place, where your LaTeX expression is inserted.\n";
+        log += "Please edit your template and add this marker.\n";
+        return [, log];
+      } else {
+        log += "\n!!! Found the old place marker '" + oldmarker + "' in your template.\n";
+        log += "Please replace it with the new marker '" + marker + "'.\n"}
+        var l = oldmarker.length;
```

Comment:
no need to re-declare `l` here with `var`

Reply:
Ah, well, a copy and paste error ... Thanks.